src/routes/dispensary/add/+page.server.ts

```diff
@@ -1,7 +1,39 @@
+import supabase from "$lib/supabase/client";
+
 export const actions = {
   create: async ({ request }) => {
-    const data = await request.formData()
-    const formName = data.get("name")
-    return { success: true, name: formName }
-  }
-}
+    try {
+      const data = await request.formData();
+      const formName = data.get("name");
+      const formAddress = data.get("address");
+      const formCity = data.get("city");
+      const formState = data.get("state");
+      const formZip = data.get("zip");
+      const formStatus = data.get("status");
+      const formCreatedBy = data.get("createdBy");
+
+      const { data: insertData, error } = await supabase
+        .from("dispensary")
+        .insert([
+          {
+            name: formName,
+            address_1: formAddress,
+            city: formCity,
+            region_id: formState,
+            postal_code: formZip,
+            validation_status_id: formStatus,
+            created_by: formCreatedBy
+          },
+        ]);
+
+      if (error) {
+        console.error("Error inserting data into supabase:", error.message);
```

Comment:
These two lines look very redundant to the behavior in the `catch` block below.  I would recommend just throwing here so it can be caught below.